Dockerfile

```diff
@@ -9,22 +9,22 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 
 # Python context setup:
-RUN pip install --upgrade pip pip-tools
+RUN pip install --no-cache-dir --upgrade pip pip-tools
 
-# ================ JAVASCRIPT
-ARG NODE_VERSION=14
-
-RUN curl -sL https://deb.nodesource.com/setup_$NODE_VERSION.x | bash -
+# ================ NODE
+# https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions
+ARG NODE_VERSION
+RUN curl -fsSL https://deb.nodesource.com/setup_$NODE_VERSION.x | bash -
 RUN apt-get install -y nodejs --no-install-recommends \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/* \
-  # smoke tests
-  && node --version \
-  && npm --version
+  && node --version
+
+# Upgrade npm itself as the included version might be old
+RUN npm install --global npm@latest && npm --version
```

Comment:
I'm fine with this, tho also the default version that comes with the specified version of node should be fine?

Reply:
On the node 14 images I was getting npm v6 and we're on v8 now. I noticed it because it complains every time we use an npm command.